pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -101,8 +101,13 @@ func (p *perfCounterNodeStatsClient) startMonitoring() error {
 		return err
 	}
 
+	networkAdapterCounter, err := newNetworkCounters()
+	if err != nil {
+		return err
+	}
+
 	go wait.Forever(func() {
-		p.collectMetricsData(cpuCounter, memWorkingSetCounter, memCommittedBytesCounter)
+		p.collectMetricsData(cpuCounter, memWorkingSetCounter, memCommittedBytesCounter, networkAdapterCounter)
```

Comment:
Is the `perfCounterUpdatePeriod` (1s) still reasonable after adding more metrics?

Reply:
`1s` should still work. And for network stats, there are some metrics per second. To ensure the cumulative data correctly, 1s should be used.